tsdb/index/tsi1/cache.go

```diff
@@ -48,11 +52,13 @@ func (c *TagValueSeriesIDCache) get(name, key, value []byte) *tsdb.SeriesIDSet {
 	if mmap, ok := c.cache[string(name)]; ok {
 		if tkmap, ok := mmap[string(key)]; ok {
 			if ele, ok := tkmap[string(value)]; ok {
+				c.tracker.IncGetHit()
```

Comment:
These are the calls that actually modify the tracked metrics. I was (mostly) able to replicate the logic 1:1 from the beta.8 code (see [here](https://github.com/influxdata/influxdb/blob/v2.0.0-beta.8/tsdb/tsi1/cache.go))